test/client/docs_test.rb

```diff
@@ -721,7 +721,7 @@
   end
 
   it "accepts a type param and does not throw an error for ES7" do
-    if !$client.version_support.es_version_7_plus?
+    if !$client.version_support.es_version_7_plus? || $client.version_support.es_version_8_plus?
       skip "This test is only needed for ES 7 onwards"
```

Comment:
```suggestion
      skip "This test is only needed for ES 7"
```

Minor word update since the skip happens for ES 8 now too.